packages/core/src/lib/implementation/persist.spec.ts

```diff
@@ -41,15 +41,7 @@ let logs: string[] = [];
 
 const resetFiles = async () => {
   vol.reset();
-  vol.fromJSON(
-    {
-      [reportPath('json')]: '',
-      [reportPath('md')]: '',
-    },
-    MEMFS_VOLUME,
-  );
-  unlinkSync(reportPath('json'));
-  unlinkSync(reportPath('md'));
+  vol.fromJSON({}, MEMFS_VOLUME);
```

Comment:
I don't understand why you use the timestamps here. The tests are using `memfs` and the virtual files are being reset in `beforeEach`. Is this change needed here? If not, this is change is adding unnecessary complexity in this case.